index.js

```diff
@@ -19,215 +19,49 @@
 // THE SOFTWARE.
 'use strict';
 
-var assign = require('object-assign');
-var alphaify = require('alphaify');
 var r = require('r-dom');
-var Immutable = require('immutable');
 var React = require('react');
 var document = require('global/document');
-var markdown = require('./markdown');
-var fs = require('fs');
-var MapGL = require('react-map-gl');
-var ScatterPlotOverlay = require('react-map-gl/src/overlays/scatterplot.react');
-/* eslint-disable max-len */
-var DraggablePoints = require('react-map-gl/src/overlays/draggable-points.react');
-/* eslint-enable max-len */
-var HeatmapOverlay = require('react-map-gl-heatmap-overlay');
-var stamenMapStyle = require('./stamen-map-style');
-var CodeSnippet = require('./code-snippet.react');
-var d3 = require('d3');
+var setLocationHash = require('set-location-hash');
 
-var initialPoints = [
-  {location: [37.79450507471435, -122.39508481737994], id: 0},
-  {location: [37.79227619464379, -122.39750244137034], id: 1},
-  {location: [37.789251178427776, -122.4013303460217], id: 2},
-  {location: [37.786862920252986, -122.40475531334141], id: 3},
-  {location: [37.78861431712821, -122.40505751634022], id: 4},
-  {location: [37.79060449046487, -122.40556118800487], id: 5},
-  {location: [37.790047247333675, -122.4088854209916], id: 6},
-  {location: [37.79275381746233, -122.4091876239904], id: 7},
-  {location: [37.795619489534374, -122.40989276432093], id: 8},
-  {location: [37.79792786675678, -122.41049717031848], id: 9},
-  {location: [37.80031576728801, -122.4109001076502], id: 10},
-  {location: [37.79920142331301, -122.41916032295062], id: 11}
-];
+var contents = {
+  'Getting Started': require('./content/getting-started/getting-started.js'),
+  'Scatterplot': require('./content/scatterplot/scatterplot.js'),
+  'Draggable Points': require('./content/draggable-points/draggable-points'),
+  'Third Party Overlays': require('./content/third-party-overlay/third-party-overlay'),
+}
 
-var pointIds = initialPoints[initialPoints.length - 1].id;
+var titles = Object.keys(contents);
 
-var Docs = React.createClass({
+var sidebar = document.getElementById('sidebar');
```

Comment:
I think we should use React to create the sidebar if only to be consistent with the the rest of the code.